pkg/runtime/config.go

```diff
@@ -247,6 +243,11 @@ func FromConfig(ctx context.Context, cfg *Config) (*DaprRuntime, error) {
 		}
 	}
 
+	// Add local scheduler address if not provided and reminders are enabled (which they are by default)
+	if intc.mode == modes.StandaloneMode && globalConfig.IsFeatureEnabled(config.SchedulerReminders) && len(intc.schedulerAddress) == 0 {
```

Comment:
For 1.15 reminders are enabled by default so shouldn't need the check

Reply:
Would this disrespect explicitly turned off reminders, further down the stack?